internal/collector/server.go

```diff
@@ -17,7 +17,7 @@ type Server struct {
 	srv      *grpc.Server
 	listener net.Listener
 
-	messages map[string]string
+	summary []*SummaryData
```

Comment:
this shouldn't be a pointer. no need for that

Reply:
![image](https://github.com/evozon/future/assets/20539991/87917440-2cd2-4692-a42e-7081315e42c8)